### PR TITLE
Line endings

### PR DESCRIPTION
**Pull request by milankoster**

### CODE REVIEW

File: src/components/Header/Header.module.scss [hunk 1]
```diff
@@ -14,22 +14,22 @@
   padding-bottom: 1.5em;
 }
 
-.Header-Separator {
```
Comment (vcaf) [inline src/components/Header/Header.module.scss:17]: I like the naming convention but just want to know how so the color and added variables are gone? were they gotten from a parent?

File: src/components/Modals/PatientEditModal/PatientEditModal.tsx [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
   const { instance } = useMsal()
   const request = useAuthRequest()
-  const [error, setError] = useState(false)
 
-  const [patientForm, setPatientForm] = useState({
-    firstName: patient?.firstName,
-    lastName: patient?.lastName,
-    birthdate: patient?.birthdate,
-  })
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (vcaf) [inline src/components/Modals/PatientEditModal/PatientEditModal.tsx:20]: Not sure if this is a comment but if it should be removed?

File: src/utils/api/calls.ts [hunk 3]
```diff
@@ -19,7 +19,7 @@ export type PatientProps = {
   id?: string
   firstName: string
   lastName: string
-  birthdate: Date
+  birthdate: string
```
Comment (vcaf) [inline src/utils/api/calls.ts:22]: we're storing date of birth as a string?

**Review by vcaf — APPROVED** [review]
Overall nice job on refactoring, LOOKS REALLY GOODDD

File: src/utils/api/calls.ts (hunk 3 above)
Comment (milankoster) [reply src/utils/api/calls.ts:22]: To my knowledge they come in as a string from the API connection and need to be manually converted to dates

File: src/components/Modals/PatientEditModal/PatientEditModal.tsx (hunk 2 above)
Comment (milankoster) [reply src/components/Modals/PatientEditModal/PatientEditModal.tsx:20]: It disables lint errors from this line. 
`  const [error, setError] = useState(false)`
Error is currently unused. We do set the error from API calls, but do not show them anywhere. As a result the 'error' variable in unused, but can be applied in the future.

**Pull request merged**